Bottom: 4c9d7342c1fa09b431ca4f825cc7a9d930746d76
Top:    407f6e7c934cd2b8cd3cc387090e26de0d321a32
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-24 21:08:32 -0400

Refresh of mpc5200-ac97

---

diff --git a/sound/soc/fsl/mpc5200_psc_ac97.c b/sound/soc/fsl/mpc5200_psc_ac97.c
index 78c06ac..480b677 100644
--- a/sound/soc/fsl/mpc5200_psc_ac97.c
+++ b/sound/soc/fsl/mpc5200_psc_ac97.c
@@ -176,7 +176,8 @@ static int psc_ac97_hw_analog_params(struct snd_pcm_substream *substream,
 		__func__, substream, params_period_size(params),
 		params_period_bytes(params), params_periods(params),
 		params_buffer_size(params), params_buffer_bytes(params),
-		params_channels(params), params_rate(params), params_format(params));
+		params_channels(params), params_rate(params),
+		params_format(params));
 
 
 	if (substream->pstr->stream == SNDRV_PCM_STREAM_CAPTURE) {
